test/core/Safe.Signatures.spec.ts

```diff
@@ -271,7 +271,7 @@ describe("Safe", () => {
             } = await setupTests();
             const compatFallbackHandler = await getCompatFallbackHandler();
             const compatFallbackHandlerAddress = await compatFallbackHandler.getAddress();
-            const signerSafe = await getSafeWithOwners([user5.address], 1, compatFallbackHandlerAddress);
+            const signerSafe = await getSafeWithOwners([user5.address], 1, ethers.ZeroAddress, "0x", compatFallbackHandlerAddress);
```

Comment:
Instead of changing this everywhere, why not add a function:

```
export function getSafeWithOwnersAndSetup(...)
```

Reply:
We already have too many functions that do very similar things. If you look into this function's implementation, it calls a function that does almost the same thing but doesn't accept all the required parameters. I could do that, but it felt wrong, while having a function that maps parameters 1:1 to the setup function felt right.

Reply:
I think even a better practice is to have the function accept an object if it has too many parameters, but that's even a bigger refactor.

Reply:
https://github.com/safe-global/safe-smart-account/issues/780